app/policies/provider_policy.rb

```diff
@@ -44,4 +48,16 @@ def suggest?
   alias_method :update?, :show?
   alias_method :build_new?, :show?
   alias_method :can_list_training_providers?, :show?
+
+private
+
+  def accredited_bodies
+    provider.accredited_bodies.map do |p|
```

Comment:
it not this one right?

https://github.com/DFE-Digital/teacher-training-api/blob/d7211042f9f2d82601f4e2bdfa82aad4f7b0a13d/app/models/provider.rb#L223-L232

``` ruby
bath = Provider.where(provider_code: "B20", recruitment_cycle: RecruitmentCycle.current_recruitment_cycle).first
bath.training_providers

# or
nsta = Provider.where(provider_code: "16Z", recruitment_cycle: RecruitmentCycle.current_recruitment_cycle).first
nsta.accrediting_providers

```

Reply:
Thats returns some random hash. Can't we work with objects? So why couldn't that method return a collection of `Provider` objects?

Reply:
Oh doesn't x.accrediting_providers or x.training_providers one of them should work

Reply:
PR has changed this code has now been deleted